src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/dashboardnotifications/claimant/DefendantResponseClaimantNotificationHandler.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.dashboardnotifications.claimant;
+
+import com.google.common.base.Strings;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.client.DashboardApiClient;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.service.DashboardNotificationsParamsMapper;
+import uk.gov.hmcts.reform.dashboard.data.ScenarioRequestParams;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackParams.Params.BEARER_TOKEN;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.CREATE_CLAIMANT_DASHBOARD_NOTIFICATION_FOR_DEFENDANT_RESPONSE;
+
+@Service
+@RequiredArgsConstructor
+public class DefendantResponseClaimantNotificationHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = List.of(CREATE_CLAIMANT_DASHBOARD_NOTIFICATION_FOR_DEFENDANT_RESPONSE);
+    public static final String TASK_ID = "GenerateClaimantDashboardNotificationDefendantResponse";
+    private final DashboardApiClient dashboardApiClient;
+    private final DashboardNotificationsParamsMapper mapper;
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
+            callbackKey(ABOUT_TO_SUBMIT), this::configureScenarioForDefendantResponse
+        );
+    }
+
+    @Override
+    public String camundaActivityId(CallbackParams callbackParams) {
+        return TASK_ID;
+    }
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    private String getScenario(CaseData caseData) {
+
+        return null;
+    }
+
+    private CallbackResponse configureScenarioForDefendantResponse(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        String authToken = callbackParams.getParams().get(BEARER_TOKEN).toString();
+        String scenario = getScenario(caseData);
+        if (!Strings.isNullOrEmpty(scenario) && caseData.isApplicant1NotRepresented()) {
```

Comment:
You have not added any Scenario in the handler? any reason

Reply:
This branch comes from the base branch for the ticket CIV-13119, which create the base handler and the related cammunda and ccd. There is no scenario as it is only the base code